libr/asm/arch/z80/z80.c

```diff
@@ -53,19 +45,24 @@ static int z80OpLength (const ut8 *buf, int len) {
 	} else {
 		type = op[buf[0]].type;
 	}
-	if (type & Z80_OP8)
+	if (type & Z80_OP8) {
 		ret++;
-	if ((type & Z80_ARG8) && !(type & Z80_ARG16))								//XXX
+	}
+	if ((type & Z80_ARG8) && !(type & Z80_ARG16)) { //XXX
 		ret++;
-	if (type & Z80_OP16)
+	}
+	if (type & Z80_OP16) {
```

Comment:
I don't see the purpose to mandate braces here (most coding styles don't mandate braces). Generality aside, in this huge PR to bring uncertainty.